lib/zendesk_apps_support/validations/manifest.rb

```diff
@@ -210,6 +211,12 @@ def invalid_type_error(manifest)
         def missing_keys_validation_error(missing_keys)
           ValidationError.new('manifest_keys.missing', missing_keys: missing_keys.join(', '), count: missing_keys.length)
         end
+
+        def framework_version_iframe_only(package, manifest)
+          if (manifest['frameworkVersion'].nil? || package.iframe_only? && manifest['frameworkVersion'] < '2.0')
```

Comment:
so should we fix this too then: https://github.com/zendesk/zendesk_app_framework/blob/6a7908f1d889f5988623045e53aab7ac75464923/lib/zaf_sdk_client.js#L29 ?